README.md

```diff
@@ -52,7 +52,7 @@ If you're planning to render the same template multiple times, you do it efficie
 tmpl, _ := mustache.ParseString("hello {{c}}")
 var buf bytes.Buffer
 for i := 0; i < 10; i++ {
-    tmpl.Render(map[string]string{"c": "world"}, &buf)
+    tmpl.FRender(&buf, map[string]string{"c": "world"})
```

Comment:
Just calling out this update to the documentation, since this use case is better supported now. As far as I can see, the old example wouldn't have worked.